beacon-chain/cache/depositcache/deposits_cache.go

```diff
@@ -162,12 +162,12 @@ func (dc *DepositCache) InsertFinalizedDeposits(ctx context.Context, eth1Deposit
 		}
 		depHash, err := d.Deposit.Data.HashTreeRoot()
 		if err != nil {
-			log.WithError(err).Error("Could not hash deposit data. Finalized deposit cache not updated.")
-			return
+			//log.WithError(err).Error("Could not hash deposit data. Finalized deposit cache not updated.")
+			return errors.Wrap(err, "Could not hash deposit data. Finalized deposit cache not updated.")
```

Comment:
Errors should be lowercase and without punctuation. A better message would be `could not hash deposit data`